rubin_sim/movingObjects/directObs.py

```diff
@@ -76,7 +76,7 @@ class DirectObs(BaseObs):
         observations (in degrees).
         Default 10 degrees.
     """
-    def __init__(self, footprint='camera', rFov=2.45, xTol=5, yTol=3,
```

Comment:
I think that I just didn't want to (bother to) search for the closest, but wanted the tolerance to be big enough to match in either case. Since the rough ephemerides are on a regular grid of times, if you search for the closest observation then you could cut the rough tolerance down in size because the time difference would be smaller .. but it was just as simple to set the rough tolerance to a larger value. Going back to update this and test the effect might be a good thing to do - it could result in fewer potential matches, thus fewer fine ephemerides being needed to calculate. Even better would be to automatically set the rough tolerance, as it currently is likely set way too large for almost all objects.

Reply:
The 0.16 is a bit magical, true. Those are intended to set the start and end times in the middle of the day, so we don't miss any actual observation times. 0.16 is the approximate offset from UTC.